CHANGELOG.rst

```diff
@@ -14,27 +14,20 @@ Changelog
 * **BACKWARDS INCOMPATIBLE:** ``Whirlpool``, ``RIPEMD160``, and
   ``UnsupportedExtension`` have been removed in accordance with our
   :doc:`/api-stability` policy.
+* **BACKWARDS INCOMPATIBLE:** :attr:`~cryptography.x509.DNSName.value`,
+  :attr:`~cryptography.x509.RFC822Name.value`, and
+  :attr:`~cryptography.x509.UniformResourceIdentifier.value` will now return
+  an :term:`A-label` string if you provided a :term:`U-label`
+  internationalized domain to the constructor. See below for additional
+  deprecations related to this change.
```

Comment:
More importantly, they return a-labels for certs with a-labels, instead of returning u-labels